clients/gen/go.sh

```diff
@@ -35,48 +30,11 @@ go_config=(
     "enumClassPrefix=true"
 )
 
-SPEC_PATH=$(realpath "$1")
-readonly SPEC_PATH
-
-if [ ! -d "$2" ]; then
-    echo "$2 is not a valid directory or does not exist."
-    exit 1
-fi
-
-OUTPUT_DIR=$(realpath "$2")
-readonly  OUTPUT_DIR
-
-# create openapi ignore file to keep generated code clean
-cat <<EOF > "${OUTPUT_DIR}/.openapi-generator-ignore"
-.travis.yml
-git_push.sh
-EOF
-
-set -ex
-IFS=','
+validate_input "$@"
 
 gen_client go \
     --package-name airflow \
     --git-repo-id airflow-client-go/airflow \
     --additional-properties "${go_config[*]}"
 
-# patch generated client to support problem HTTP API
-# this patch can be removed after following upstream patch gets merged:
-# https://github.com/OpenAPITools/openapi-generator/pull/6793
```

Comment:
@houqp I see that this patch has been merged in openapi, so I think it should be safe to remove it now, though I'm getting some testcase failures and hoping that it's not related to this change.

Reply:
i think you would need to bump `OPENAPI_GENERATOR_CLI_VER` to the latest version to pick up the fix.

Reply:
also did you see test cases failing for go client or python client?

Reply:
Thanks @houqp, used the latest stable version of openapi gen cli. With that, I think we should bump up the go client version from 1.0.0 to 1.1.0? as I see some changes in the code structure/imports due to that.

Reply:
yep, we should bump client version for changes from code gen. whether it should be `1.1.0` or `1.0.1` depends on whether the new client as new feature or not.

Reply:
I believe there would be an addition of some more APIs, and a little bit of diff import structure as well, so we should go with `1.1.0` at least.

Reply:
sounds good 👍